docs/logql.md

```diff
@@ -228,41 +235,67 @@ These logical/set binary operators are only defined between two vectors:
 - `or` (union)
 - `unless` (complement)
 
-`vector1 and vector2` results in a vector consisting of the elements of vector1 for which there are elements in vector2 with exactly matching label sets. Other elements are dropped.
+`vector1 and vector2` results in a vector consisting of the elements of vector1 for which there are elements in vector2 with exactly matching label sets.
+Other elements are dropped.
 
 `vector1 or vector2` results in a vector that contains all original elements (label sets + values) of vector1 and additionally all elements of vector2 which do not have matching label sets in vector1.
 
-`vector1 unless vector2` results in a vector consisting of the elements of vector1 for which there are no elements in vector2 with exactly matching label sets. All matching elements in both vectors are dropped.
+`vector1 unless vector2` results in a vector consisting of the elements of vector1 for which there are no elements in vector2 with exactly matching label sets.
+All matching elements in both vectors are dropped.
 
 ##### Examples
 
-This contrived query will return the intersection of these queries, effectively `rate({app="bar"})`
+This contrived query will return the intersection of these queries, effectively `rate({app="bar"})`:
 
-> `rate({app=~"foo|bar"}[1m]) and rate({app="bar"}[1m])`
+```logql
+rate({app=~"foo|bar"}[1m]) and rate({app="bar"}[1m])
+```
 
 #### Comparison operators
 
--  `==` (equality)
--  `!=` (inequality)
--  `>` (greater than)
--  `>=` (greater than or equal to)
--  `<` (less than)
--  `<=` (less than or equal to)
+- `==` (equality)
+- `!=` (inequality)
+- `>` (greater than)
+- `>=` (greater than or equal to)
+- `<` (less than)
+- `<=` (less than or equal to)
 
-Comparison operators are defined between scalar/scalar, vector/scalar, and vector/vector value pairs. By default they filter. Their behavior can be modified by providing bool after the operator, which will return 0 or 1 for the value rather than filtering.
+Comparison operators are defined between scalar/scalar, vector/scalar, and vector/vector value pairs.
+By default they filter.
+Their behavior can be modified by providing `bool` after the operator, which will return 0 or 1 for the value rather than filtering.
 
-Between two scalars, the bool modifier must be provided and these operators result in another scalar that is either 0 (false) or 1 (true), depending on the comparison result.
+Between two scalars, these operators result in another scalar that is either 0 (false) or 1 (true), depending on the comparison result.
+The `bool` modifier must **not** be provided.
```

Comment:
Not sure about this sentence.
The original looked plain and simply wrong, so I suggest this wording.

Reply:
Thanks! I just checked and this should actually state that between two scalars, the `bool` modifier is ignored.